src/components/settings-drawer.tsx

```diff
@@ -136,17 +155,19 @@ const useStyles = makeStyles((theme) => ({
   legendLabel: {
     '@media (min-aspect-ratio: 8/10)': {
       fontSize: theme.typography.h6.fontSize,
-      padding: theme.spacing(1),
+      paddingBottom: theme.spacing(1),
+      textAlign: 'center',
     },
     '@media (max-aspect-ratio: 8/10)': {
       fontSize: theme.typography.h6.fontSize,
       padding: theme.spacing(1),
+      textAlign: 'center',
     },
   },
   trajGroup: {
     '@media (min-aspect-ratio: 8/10)': {
       flexDirection: 'row',
-      paddingLeft: theme.spacing(4),
+      paddingLeft: theme.spacing(8),
     },
     '@media (max-aspect-ratio: 8/10)': {
       flexDirection: 'row',
```

Comment:
Can you add more vertical space between the trajectories group? I think they are too close to each other

![image](https://user-images.githubusercontent.com/11761240/93631321-5b0bfe00-f9b9-11ea-841b-911d1b372dee.png)